tracer/build/_build/Build.Steps.cs

```diff
@@ -769,6 +771,16 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
                 workingDirectory / versionedName);
         });
 
+    Target ZipMonitoringHomeOsx => _ => _
+        .Unlisted()
+        .After(BuildTracerHome, BuildNativeLoader)
+        .OnlyWhenStatic(() => IsOsx)
+        .Requires(() => Version)
```

Comment:
I don't think this is necessary?

```suggestion
```

Reply:
done in [7f42b05](https://github.com/DataDog/dd-trace-dotnet/pull/3863/commits/7f42b05e794ac84b30e5fc059d4d6c8c7b8eb4af)